php_redis.h

```diff
@@ -260,9 +260,17 @@ PHP_REDIS_API int redis_sock_read_multibulk_multi_reply_loop(
     INTERNAL_FUNCTION_PARAMETERS, RedisSock *redis_sock, zval *z_tab, 
     int numElems);
 
-#ifndef _MSC_VER
 ZEND_BEGIN_MODULE_GLOBALS(redis)
+    int lock_release_lua_script_uploaded;
+    char lock_release_lua_script_hash[41];
```

Comment:
Why this is not just a pointer?

Reply:
To avoid the need for later cleanup when process ends.
Do you see advantages in using a pointer?

Reply:
Sounds reasonable

Reply:
In the end I switched to pointer, because your suggestion with using estrdup brings much more simplicity and is only possible with pointer.